src/app/features/tasks/add-task-bar/add-task-bar.component.ts

```diff
@@ -177,6 +188,12 @@ export class AddTaskBarComponent implements AfterViewInit, OnDestroy {
 
   onBlur(ev: FocusEvent) {
     const relatedTarget: HTMLElement = ev.relatedTarget as HTMLElement;
+    if (!relatedTarget || (relatedTarget &&
+      !relatedTarget.className.includes('close-btn') &&
+      !relatedTarget.className.includes('switch-add-to-btn')))
+    {
+      this._taskService.setSavedTodo(this.taskSuggestionsCtrl.value);
```

Comment:
I'd probably use the sessionStorage instead. The task service is already pretty big and more focused on actual model stuff. 

If you change this, it would be cool, if you could put the key for that into this file:
`src/app/core/persistence/ls-keys.const.ts`